src/DetailsPage.js

```diff
@@ -18,8 +19,25 @@ export default class DetailsPage extends Component {
         console.log(this.state.pokemon)
         return (
             <>
+            <header>
+                <NavLink
+                to="/"
+                activeStyle={{
+                fontWeight: "bold",
+                color: "turquoise"
+            }}
+            >
+            Home
+                </NavLink>
+              </header>
             <div>
```

Comment:
Everything that is happening on lines 33-41 need to be refactored into the PokeDetails component.